src/map/GMap.js

```diff
@@ -29,7 +29,7 @@
         this._prevMin = { x: 0, y: 0 };
         this._prevMax = { x: 0, y: 0 };
     }
-    Overlay.prototype = new google.maps.OverlayView();
+    Overlay.prototype = new google.maps.OverlayView() || null;
```

Comment:
I was suppose to remove that mod ... I'm trying to figure out how to fail
gracefully

On Wednesday, October 14, 2015, Gordon Smith <notifications@github.com>
wrote:

> In src/map/GMap.js
> <https://github.com/hpcc-systems/Visualization/pull/1041#discussion_r42018256>
> :
>
> > @@ -29,7 +29,7 @@
> >          this._prevMin = { x: 0, y: 0 };
> >          this._prevMax = { x: 0, y: 0 };
> >      }
> > -    Overlay.prototype = new google.maps.OverlayView();
> > +    Overlay.prototype = new google.maps.OverlayView() || null;
>
> || null makes no sense...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpcc-systems/Visualization/pull/1041/files#r42018256>.
>